packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -61,6 +62,30 @@ const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( {
 	);
 };
 
+interface TrackDesignViewProps {
+	categorization?: string | null;
+	design: Design;
+	isPremiumThemeAvailable?: boolean;
+}
+
+const TrackDesignView: React.FC< TrackDesignViewProps > = ( {
```

Comment:
I am curious about this new component `TrackDesignView`, do you mind giving some insights on why we need it here? Thanks